projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/__tests__/transform-job-to-notebook-component.spec.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import TransformJobToNotebookDialog from '../components/TransformJobToNotebook';
+import { render, fireEvent } from '@testing-library/react';
+import { jobData } from '../jobData';
+import { VdkOption } from '../vdkOptions/vdk_options';
+import { IJobPathProp } from '../components/props';
+
+const defaultProps: IJobPathProp = {
+  jobPath: 'test-path'
+};
+
+// created with the expectation to compare a rendered value with expected value parsed from config.ini
+// yet to be implemented
+describe('#render()', () => {
+  it('should return contain job path input with placeholder equal to jobPath from props', () => {
+    const component = render(
+      new TransformJobToNotebookDialog(defaultProps).render()
```

Comment:
We kind of mentioned this during the demo, but we should probably rename this to "Convert Job To Notebook" because "Transform Job" is semantics we've already used and can be a cause of confusion.

Reply:
Renamed it.